Header header
string Configuration_name
int32 PT_mode
float32 Acquisition_frequency
int32 Frames_per_bloc
int32 Blocs_per_PT
float32 Pressure_for_auto_start
float32 Pressure_difference_for_auto_stop
bool Result_sending
bool Save_synthetic_data_for_delayed_request
int32 Limit_lpm_detection_size
bool Save_images
int32 Vignetting_lower_limit_size
float32 Appendices_ratio
float32 Interval_for_measuring_background_noise
int32 Image_nb_for_smoothing
bool Analog_output_activation
float32 Gain_for_analog_out
int32 Minimum_object_number
float32 Maximal_internal_temperature
string Operator_email
int32 SD_card_remaining_memory
